RequestManagementSystem/Client/ReqClient.py

```diff
@@ -304,17 +305,28 @@ def finalizeRequest(self, requestID, jobID, useCertificates=True):
 
     # The request is 'Done', let's update the job status. If we fail, we should re-try later
     monitorServer = RPCClient("WorkloadManagement/JobMonitoring", useCertificates=useCertificates)
-    res = monitorServer.getJobPrimarySummary(int(jobID))
+    res = monitorServer.getJobSummary(int(jobID))
```

Comment:
I think all that should not be in the `ReqClient`. Rather, a calback should be done to whatever job service, saying "I executed all the operations for failover of job XYZ", and then what is done with this information should remain to the internal job machinery.

Reply:
Any proposal on how to concretely do that? 
Can we first see whether the logic is correct before embarking on that?

Reply:
I'll think of a proposal on how to do that.
as for the logic itself, I would have to understand first myself :D

Reply:
For another PR then.